samples/android/non-android-lib/build.gradle

```diff
@@ -0,0 +1,19 @@
+apply plugin: 'kotlin'
```

Comment:
why do we put this into `samples/android/non-android` instead of `samples/kotlin`?

Reply:
Still related to android, this shows that spek (ide plugin) can still be used in a non-android library within an android project.